api/v1/service/resource/resource.go

```diff
@@ -353,6 +354,7 @@ func initResource(r model.Resource) *resource.ResourceData {
 		WebURL:              lv.URL,
 		RawURL:              getStringReplacer(lv.URL, r.Catalog.Provider).Replace(lv.URL),
 		HubURLPath:          fmt.Sprintf("%s/%s/%s/%s", r.Catalog.Name, r.Kind, r.Name, lv.Version),
+		HubRawURLPath:       fmt.Sprintf("/resource/%s/%s/%s/raw", r.Catalog.Name, strings.ToLower(r.Kind), r.Name),
```

Comment:
I think there should be version in the latestVersion struct 
i.e. resource/tekton/task/git-clone/0.9/raw